app/views/events/_below_table.html.haml

```diff
@@ -13,4 +13,88 @@
                        id: 'bulk_email',
                        data: { confirm: t('notifications_module.send_unspecified_notification_confirm', count: @num_of_matching_events) }, 
                        class: 'danger'
+     
+    - if policy(@conference).manage?
+      %p
+        = link_to t('events_module.bulk_edit.toggle'), "#bulk_edit", data: { function: 'toggle', args: { target: '#bulk_edit' } }
```

Comment:
The whole interface looks a little rough. I wonder why each dropdown needs it's own Set button, when you can easily just make the whole thing a form and then submit all changes at once. It can even show what to edit in the dropdown as first placeholder text:

[ Bulk set state ] [ Bulk set track ] [ Bulk set type ] [ GO ]